### PR TITLE
Bump SymbolicUtils.jl to 0.19

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
I'm having some trouble isolating the issue due to FromFile.jl eval. But it seems like it's:

```julia
using Metatheory
rhs = :(~(x^-1))
Metatheory.Syntax.makeconsequent(rhs)
[9:50 PM] julia> Metatheory.Syntax.makeconsequent(rhs)
ERROR: Error when parsing right hand side
Stacktrace:
 [1] error(s::String)
   @ Base .\error.jl:33
 [2] makeconsequent(expr::Expr)
   @ Metatheory.Syntax C:\Users\accou\.julia\dev\Metatheory\src\Syntax.jl:64
 [3] top-level scope
   @ REPL[2]:1
```

@shashi

### CODE REVIEW

**Comment by MilesCranmer** [comment]
Thanks for the PR Chris. I think this PR also needs to revert this particular commit in master: https://github.com/MilesCranmer/SymbolicRegression.jl/commit/3b2608da21a0c9c57282cf40feac7649ef9849d9 to get the newer SymbolicUtils.jl working again, since the data types are quite different for `>0.6`. The other issue I think is with the simplification backend which we discussed on #59–we will need to automatically convert exponents into repeated multiplication when converting back to SymbolicRegression.jl.

**Comment by MilesCranmer** [comment]
Another option may be to just re-implement the export functionality from scratch to exclusively do exports and nothing else? This would be much less work I think.

I have the SymbolicUtils.jl simplification utilities turned off by default anyways since I noticed they were taking an exponentially longer amount of time for very complex expressions, so we could simply remove it entirely for now, and just leave a simple export functionality. The simplification functionality was a bit overkill anyways to have within a genetic algorithm since it will try to simplify the entire expression every single time called, when in reality I would want something a bit more stochastic/lightweight to partially simplify only part of an expression every time it is called.

**Comment by MilesCranmer** [comment]
@ChrisRackauckas FYI I switched to a normal `include()`-based system so it should be easier for you to use normal code analysis tools to find this.

**Comment by ChrisRackauckas** [comment]
@shashi wasn't something like `@variables x::SafeFloat` added for removing some of the simplifications? That might be what's needed here.

**Comment by MilesCranmer** [comment]
I think it's easier if we leave out the simplification for now and add it back in later. For the simplification to work i realized I need to experiment with some sort of "max_evals" to the simplifier, so that very complex expressions don't slow down the overall search too much.

The edits I pushed yesterday are basically removing the simplification parts, and simply leaving in the import/export utils, so it can interface with SymbolicUtils.

**Comment by shashi** [comment]
`@variables x::LiteralReal`, there is also `SafeReal` which is to avoid cancellation in division only.

**Comment by MilesCranmer** [comment]
Seems like it's working now which is good. I'm going to try to get 100% test coverage on the SymbolicUtils interface before merging, just in case.

File: src/InterfaceSymbolicUtils.jl [hunk 1]
```diff
@@ -4,12 +4,108 @@ using SymbolicUtils
...
+const SUPPORTED_OPS = (cos, sin, exp, cot, tan, csc, sec, +, -, *, /)
+
+isgood(x::SymbolicUtils.Symbolic) = SymbolicUtils.istree(x) ? all(isgood.([SymbolicUtils.operation(x);SymbolicUtils.arguments(x)])) : true
+subs_bad(x) = isgood(x) ? x : Inf
+
+function parse_tree_to_eqs(tree::Node, options::Options, index_functions::Bool=false)
+    if tree.degree == 0
+        # Return constant if needed
+        tree.constant && return subs_bad(tree.val)
+        return SymbolicUtils.Sym{Number}(Symbol("x$(tree.feature)"))
```
Comment (MilesCranmer) [inline src/InterfaceSymbolicUtils.jl:18]: This one should be `LiteralReal` instead of `Number`. Same for other instances.

**Comment by MilesCranmer** [comment]
Cool, `LiteralReal` seems to work - thanks @shashi!

```julia
using SymbolicUtils, SymbolicRegression
options = Options(binary_operators=(+, *))
tree = Node("x1") * Node("x1")
eqn = convert(Symbolic, tree, options)
tree_copy = convert(Node, eqn, options)
println(tree_copy)  # =(x1 * x1)
```
Before, this would give an error as it tries to convert it to `x1^2`, while `^` is not specified as an operator. But now it works and doesn't do simplification.

**Review by MilesCranmer — APPROVED** [review]

**Comment by MilesCranmer** [comment]
Thanks for starting this PR, happy to have this finally working!

**Comment by MilesCranmer** [comment]
See https://github.com/SciML/DataDrivenDiffEq.jl/pull/346 for the update to DataDrivenDiffEq.jl

**Comment by ChrisRackauckas** [comment]
Awesome, thanks for completing this.

**Pull request merged**